libs/search-category/README.md

```diff
@@ -0,0 +1,10 @@
+# @daffodil/search-category
+
+An optional package which adds the related categories feature onto the `@daffodil/category` package.
```

Comment:
Docs.

Reply:
fixed the root readmes for all the added packages